### PR TITLE
Fix back to process button

### LINKED ISSUE #856: Metrics screen: "go back to process" button is not distinctive
On the Metrics screen the "go back to process" button is easily confused with "go to main page" link

Zrzut ekranu z 2020-04-08 17-49-03

### PR DESCRIPTION
**Pull request by lciolecki**
This resolved #856 and #238.

### CODE REVIEW

File: ui/client/containers/NussknackerApp.tsx [hunk 1]
```diff
@@ -42,25 +59,17 @@ export class EspApp extends React.Component {
...
     const match = this.getMetricsMatch()
     if (this.canGoToProcess()) {
-      return (
-        <div className="top-left-button">
-          <ProcessLink processId={match.params.processId}>
-            <span className="glyphicon glyphicon-menu-left"/>
-          </ProcessLink>
-        </div>
-      )
+      return (<ProcessBackButton processId={match?.params.processId}/>)
```
Comment (mproch) [inline ui/client/containers/NussknackerApp.tsx:72]: why there is one "?"? in canGoToProcess we check that it's not null?

File: ui/client/containers/NussknackerApp.tsx [hunk 2]
```diff
@@ -25,10 +24,28 @@ import ErrorHandler from "./ErrorHandler"
...
+    processId: string,
+  },
+}
+
+class NussknackerApp extends React.Component<Props, State> {
+
+  private readonly mountedHistory: UnregisterCallback
+
+  constructor(props) {
+    super(props)
```
Comment (mproch) [inline ui/client/containers/NussknackerApp.tsx:47]: is componentDidMount the same as constructor in our case?

File: ui/client/components/Process/ProcessBackButton.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  const process = processName || processId
+  const {t} = useTranslation()
+
+  return (
+    <div className={"back-to-process-button"}>
+      <ProcessLink processId={processId}>
+        <ButtonWithFocus className="back-to-process-button">
+          <ProcessBackIcon/>
+          <span className="button-text">
+            {t("processBackButton.text", "back to process {{process}}", {process})}
```
Comment (mproch) [inline ui/client/components/Process/ProcessBackButton.tsx:23]: Back to process {{...?

File: ui/client/containers/NussknackerApp.tsx (hunk 2 above)
Comment (lciolecki) [reply ui/client/containers/NussknackerApp.tsx:50]: It works as same as before .. So I guess it's ok. I've tested app without this assign and application did not crash..

File: ui/client/components/Process/ProcessBackButton.tsx (hunk 3 above)
Comment (lciolecki) [reply ui/client/components/Process/ProcessBackButton.tsx]: On mock was small letter.. ;) But I've changed it to big one.

**Review by mproch — APPROVED** [review]

**Pull request merged**